lib/src/base-styles/_nx-btn.scss

```diff
@@ -101,6 +101,34 @@
       height: 16px;
     }
   }
+
+  &--small {
+    font-size: var(--nx-font-size-s);
+    padding: var(--nx-spacing-base) var(--nx-spacing-3x);
+    margin: 0 var(--nx-spacing-3x) 0 0;
```

Comment:
I'm not seeing a spec for this in the mockup, where did you get it from?

Reply:
removed. https://github.com/sonatype/sonatype-react-shared-components/pull/1220/commits/d6eeafa238b8da4d8e0ff5f0725ffb2148206c44
the mockup show a spacing between text and icon. but the default style handle that already